### PR TITLE
docs: add example for startActivity

### PR DESCRIPTION
**Pull request by KazuCocoa**
for https://github.com/appium/appium/issues/20730#issuecomment-2463928490

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -878,6 +878,25 @@ flags | string | no | Intent startup-specific flags as a hexadecimal string. Che
 
 The actual stdout of the downstream `am` command.
 
+#### Command Example
+
+The example command below generates `shell am start-activity -W -n io.appium.android.apis/io.appium.android.apis.ApiDemos -S -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000'` which is called in a new session request.
+If you replace `'io.appium.android.apis/io.appium.android.apis.ApiDemos'` with your expcted launchable activity, it will be the alternative method to start the activity in session.
```
Comment (mykola-mokhnach) [inline README.md:884]: typo `expected`

Comment (mykola-mokhnach) [reply README.md:884]: it will be -> it would be

File: README.md [hunk 2]
```diff
@@ -878,6 +878,25 @@ flags | string | no | Intent startup-specific flags as a hexadecimal string. Che
 
 The actual stdout of the downstream `am` command.
 
+#### Command Example
```
Comment (mykola-mokhnach) [inline README.md:881]: Just `Examples`

File: README.md [hunk 3]
```diff
@@ -878,6 +878,25 @@ flags | string | no | Intent startup-specific flags as a hexadecimal string. Che
...
+
+The example command below generates `shell am start-activity -W -n io.appium.android.apis/io.appium.android.apis.ApiDemos -S -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000'` which is called in a new session request.
```
Comment (mykola-mokhnach) [inline README.md:883]: Use the below code snippet to represent the following shell command ...

**Comment by KazuCocoa** [comment]
Updated

File: README.md [hunk 4]
```diff
@@ -878,6 +878,25 @@ flags | string | no | Intent startup-specific flags as a hexadecimal string. Che
 
 The actual stdout of the downstream `am` command.
 
+#### Example
+
+Use the code snippet below to represent the following shell command: `shell am start-activity -W -n io.appium.android.apis/io.appium.android.apis.ApiDemos -S -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000'`, which is called in a new session request.
```
Comment (mykola-mokhnach) [inline README.md:883]: which is usually similar to what is called in a new session request

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**